scripts/importDpsConstants.ts

```diff
@@ -84,37 +86,29 @@ if (method === 'errorCodes') {
     fs.writeSync(outputFile, `${constant.description} = ${constant.code},\n`)
   })
   fs.writeSync(outputFile, '}\n')
-} else if (method === 'types') {
-  // types are string constants but have extra info
+} else {
+  // other constants are strings with extra info
 
   fs.writeSync(outputFile, `/** ${documentation} */\n`)
   fs.writeSync(outputFile, `export const ${method} = [\n`)
-  constants.forEach((constant: TypeConstant) => {
-    fs.writeSync(outputFile, `/** ${constant.description} */\n`)
+  constants.forEach(constant => {
     fs.writeSync(
       outputFile,
-      `{ code: ${JSON.stringify(constant.code)}, active: ${constant.active}, nomisCode: ${JSON.stringify(constant.nomisCode)} },\n`,
+      `{ code: ${JSON.stringify(constant.code)}, description: ${JSON.stringify(constant.description)},\n`,
     )
+    if ('active' in constant) {
+      fs.writeSync(outputFile, `active: ${constant.active}, nomisCode: ${JSON.stringify(constant.nomisCode)} },\n`)
+    } else {
+      fs.writeSync(outputFile, '},\n')
+    }
```

Comment:
I liked the simpler version before, without the if/else and with `"active"` always there 😬

Reply:
that's not simpler if `active` is made up and always true as defined in the _ui_ app and not the _api_ where this data is supposed to be sourced from